spec/controllers/criteria_controller_spec.rb

```diff
@@ -1031,4 +1031,52 @@
       let(:params) { { course_id: course.id, assignment_id: assignment.id } }
     end
   end
+
+  context '#update_position' do
```

Comment:
This should be `describe` instead of `context` (we use `describe` to introduce a test block for a particular method). Also, the method is called `update_positions`, not `update_position`.

However, I notice that there is already a test for `update_positions` up above on on line 740; please move the tests you added there. You can use the `describe` line as the outer block, and nest the existing test underneath it, together with the new ones you're adding. This way we keep all of the tests for the same method together.